tests/system/large/test_remote_function.py

```diff
@@ -1432,9 +1433,7 @@ def square_num(x):
 
 
 def test_remote_function_via_session_vpc_invalid(session):
```

Comment:
TODO: Might be better to replace this test with a unit test where we verify that we're actually sending VPC info in the relevant requests.

Reply:
Removed this test, as it's redundant with the check here: https://github.com/googleapis/python-bigquery-dataframes/blob/72e95834f8755760f3529d38f340703f3b971f0a/tests/system/large/test_remote_function.py#L1426